lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -168,8 +168,10 @@ describe('CategoryRenderer', () => {
     assert.ok(Number.isInteger(scoreInDom) && scoreInDom > 10, 'category score is rounded');
     assert.equal(title.textContent, category.title, 'title is set');
 
+    // Audits in 'hidden' group are not rendered.
```

Comment:
can you add a test to `category-renderer-test` and `pwa-category-renderer-test` for an explicit check that the hidden audits aren't being rendered. Basically like the `manual` tests, but instead that the audits _aren't_ in the report.